autopush/endpoint.py

```diff
@@ -254,6 +273,49 @@ def _db_error_handling(self, d):
         d.addErrback(self._response_err)
         return d
 
+    def _extract_jwt(self, token, crypto_key):
```

Comment:
Nothing in this function uses `self`. The function should probably reside elsewhere, maybe utils.py.